.github/workflows/zero-downtime-tests.yml

```diff
@@ -42,7 +47,7 @@ jobs:
         include:
           - CLOUD_PROVIDER: "GKE"
             PLATFORM_VERSION: "1.22"
-            CLUSTER_NAME: "keptn-integration-test-2"
+            CLUSTER_NAME: "keptn-ZD-test-2"
```

Comment:
please leave this as it was, since we are using the integration test cluster for now